src/views/ConfirmEmailView.vue

```diff
@@ -22,6 +22,16 @@
 
         <ResendConfirmationEmailModal :default-email="this.emailAddress" />
 
+        <b-button
+          size="sm"
+          variant="link"
+          class="pl-0 pr-0"
+          @click.prevent="confirmEmail"
+          v-if="showManualConfirm"
```

Comment:
This `v-if` should probably also check if `this.emailAddress` is set, although it's probably not super necessary.

```suggestion
          v-if="showManualConfirm && this.emailAddress"
```